CONTRIBUTION.rst

```diff
@@ -0,0 +1,532 @@
+« Et voilà : ».
+
+=> Traduisez ``mot deux-points deux-points`` par
+``mot espace-insécable deux-points deux-points``.
+
+Pour saisir une espace insécable faites : « Compose » + « espace » ×2
+
+Outils utiles pour la traduction
+--------------------------------
+
+Potodo
+~~~~~~
+
+| Permet de d'identifier les parties de la documention qu'il reste à traduire.
+| Installez-le à l'aide de *pip* (``pip install potodo``) dans un environnement
+  ``python3.6`` ou plus.
+| `Lien vers le repository <https://github.com/seluj78/potodo>`__
+
+Pogrep
+~~~~~~
+
+| Permet de rechercher dans la documention des termes. Utiles si on a un doute
+  sur comment traduire un terme ou chercher la traduction d'un terme dans
+  d'autres fichiers.
+| Installez-le à l'aide de *pip* (``pip install pogrep``) dans un
+  environnement.
+| `Lien vers le repository <https://github.com/JulienPalard/pogrep>`__
+
+Padpo (beta)
+~~~~~~~~~~~~~~~
+
+| Analyseur de code qui vérifie la grammaire et l'orthographe et la syntaxe
+  du fichier .po.
+| Installez-le à l'aide de *pip* (``pip install padpo``) dans un environnement
+  ``python3.7`` ou plus.
+| `Lien vers le repository <https://github.com/vpoulailleau/padpo>`__
+
+Powrap
+~~~~~~
+
+| Formateur de fichier .po.
+| Installez-le à l'aide de *pip* (``pip install powrap``) dans un
+  environnement.
+| `Lien vers le repository <https://github.com/JulienPalard/powrap>`__
+
+Ressources de traduction
+-------------------------
+
+- le canal IRC `#python-fr <http://irc.lc/freenode/python-fr>`_ sur freenode ;
+- la `liste traductions AFPy <http://lists.afpy.org/mailman/listinfo/traductions>`_ ;
+- la `liste de diffusion doc-sig
+  <https://mail.python.org/mailman/listinfo/doc-sig>`_ ;
+- les `glossaires et dictionnaires de traduc.org
+  <https://traduc.org/Glossaires_et_dictionnaires>`_, en particulier le
+  `grand dictionnaire terminologique <http://gdt.oqlf.gouv.qc.ca/>`_
+  de l'Office québécois de la langue française ;
+- le `glossaire Python <https://docs.python.org/fr/3/glossary.html>`_, car
+  il est déjà traduit ;
+- le `guide stylistique pour le français de localisation des produits Sun
+  <https://web.archive.org/web/20160821182818/http://frenchmozilla.org/FTP/TEMP/guide_stylistique_December05.pdf>`_ donne
+  beaucoup de conseils pour éviter une traduction trop mot à mot ;
+- `deepl.com/translator <https://www.deepl.com>`_ ;
+- `Petites leçons de typographie <https://jacques-andre.fr/faqtypo/lessons.pdf>`_,
+  résumé succint de typographie, utile pour apprendre le bon usage des
+  majuscules, des espaces, etc.
+
+Glossaire
+---------
+
+Afin d'assurer la cohérence de nos traductions, voici quelques propositions et
+rappels pour les termes fréquents à traduire, n'hésitez pas à ouvrir un ticket
+si vous n'êtes pas d'accord.
+
+Pour trouver facilement comment un terme est déjà traduit dans notre documentation,
+vous pouvez utiliser
+`find_in_po.py <https://gist.github.com/JulienPalard/c430ac23446da2081060ab17bf006ac1>`_.
```

Comment:
Soon, y'aura une commande comme powrap etc... cc @JulienPalard

Reply:
C'est pogrep ? Je n'ai pas relu cette section là et ai fais un bête copy/paste.
Je peux remplacer cette ligne par la mention de pogrep, qui me semble plus pertinente, pour un meilleur résultat.

Reply:
En effet c'est pogrep maintenant : pypi.org/p/pogrep

Reply:
@JulienPalard Ok noté